src/components/filter.jsx

```diff
@@ -1,58 +1,66 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Dropdown, Form, Button, Row, Col } from 'bootstrap-4-react';
+import './style.css';
+import { Dropdown, Button } from 'bootstrap-4-react';
 
 class filter extends Component {
   state = {
-    content: '',
+    pin: '',
+    name: '',
   };
 
-  handleChange = e => {
+  handleNameChange = e => {
     this.setState({
-      content: e.target.value,
+      name: e.target.value,
     });
   };
 
-  handleSubmit = e => {
+  handlePinChange = e => {
+    this.setState({
+      pin: e.target.value,
+    });
+  };
+
+  handleSubmitName = e => {
     e.preventDefault();
-    this.props.pin(this.state.content);
-    this.setState({ content: '' });
```

Comment:
This should be present in `handleSubmitName( )` and `handleSubmitPIN( )`

Reply:
'content' state property is being replaced by 'name'. Hence she has already added this in handleSubmitPIN( ) as  this.props.pin(this.state.name);